Directory.Build.props

```diff
@@ -80,4 +80,7 @@
     <!-- Maps paths injected by CallerFilePath attribute to be related and independent from repository location on hard drive -->
     <PathMap>$(MSBuildThisFileDirectory)=/</PathMap>
   </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Internal.Analyzers " VersionOverride="2.9.7" IncludeAssets="analyzers" PrivateAssets="all" />
```

Comment:
How will this be updated?
Please add a comment with some (at least vague) detail about the constraint.